build/import/GenerateThisAssemblySourceFile.targets

```diff
@@ -1,44 +0,0 @@
-<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
```

Comment:
This file is no longer necessary as all the properties (other than `FullName`) are generated by Nerdbank.GitVersioning. See ExportRuleAttribute.cs for the FullName workaround.